src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +132,72 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
+    """Observes births and births with neural tube defects. Output can be stratified
+    by year and by sex.
+    """
+    configuration_defaults = {
+        'metrics': {
+            'live_births_with_ntds': {
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    @property
+    def name(self):
+        return f'live_birth_with_ntd_observer'
+
+    def setup(self, builder):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.config = builder.configuration['metrics']['live_births_with_ntds'].to_dict()
+        self.live_birth_counts = Counter()
+        self.with_disease = Counter()
+        tmp = builder.configuration['time']['start']
+        self._step_start_time = pd.Timestamp(tmp['year'], tmp['month'], tmp['day'])
+
+        columns_required = ['alive', f'{self.disease}', 'entrance_time']
+        if self.config['by_sex']:
+            columns_required.append('sex')
+        self.population_view = builder.population.get_view(columns_required)
+
+        builder.value.register_value_modifier('metrics', self.metrics)
+        builder.event.register_listener('collect_metrics', self.on_collect_metrics)
+
+    def on_collect_metrics(self, event):
+        pop = self.population_view.get(event.index)
+        self.live_birth_counts.update(
+            get_live_births(pop, self.config, self._step_start_time, event.time.year))
```

Comment:
Do these in two lines:
```
live_births_this_step = get_live_births(...)
self.live_birth_counts.update(live_births_this_step).
```
Two reasons:
1. It's easier to use a debugger if get_live_births breaks or (the worse case) you are unintentionally overwriting keys.
2. Our pattern for stratification is to get the counts this step, update the dictionary keys with our stratification criteria, then update the observer count dict, so we'll have to change it anyway.